vite-plugin/src/index.ts

```diff
@@ -78,6 +85,13 @@ export default function jahia(
          */
         fileName?: string;
       };
+      /**
+       * Enable source maps for the server-side bundle.
+       *
+       * @default `true`
```

Comment:
Out of curiosity, what's the rationality behind having a different default value between client and server? 🤔

Reply:
It's ok to have source maps by default on the server, on the client I'm not so sure. Maybe `false` is a better default value for the client build?

Reply:
Better idea suggested during refinement: true in `yarn dev` mode, false in `yarn build` mode